_topic_maps/_topic_map.yml

```diff
@@ -2706,6 +2706,8 @@ Topics:
     File: monitoring-overview
   - Name: Configuring the monitoring stack
     File: configuring-the-monitoring-stack
+  - Name: Common monitoring configuration scenarios
```

Comment:
Should this go before "Configuring the monitoring stack"? It might be helpful to consider potential configuration scenarios before jumping into the details about how to do the configuration.

Reply:
Good suggestion. I agree and will move it before "Configuring the monitoring stack"